config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :template
```

Comment:
when you do `be rake routes` this will allow it to show every path (even ones we aren't implementing) , but you can limit that by doing:
`resources :template, only: [:index, :create]`

Reply:
yes, true. We should change this soon;  not sure if it needs to happen in this PR, tho.